cypress/support/validations/undefinedValidation.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @module undefinedValidation
+ * @function undefinedValidation
+ * @description To validate the content of the response if undefined
+ * @param {Object} validationTypeObject - The object consists of required fields for validation.
+ * @param {Object} apiOrEventObject - Extracted API or Event object from global list.
+ * @param {String} validationType - To check whether to validate event or method
+ * @example
+ * cy.undefinedValidation(validationTypeObject, apiOrEventObject, validationType)
+ */
+
+const CONSTANTS = require('../constants/constants');
+const { fireLog } = require('../cypress-support/src/utils');
+
+Cypress.Commands.add(
+  'undefinedValidation',
+  (validationTypeObject, apiOrEventObject, validationType) => {
+    if (validationTypeObject != null && apiOrEventObject != null) {
+      try {
+        const methodOrEventResponse =
+          validationType == CONSTANTS.EVENT
+            ? apiOrEventObject.eventResponse
+            : validationType == CONSTANTS.METHOD
+              ? apiOrEventObject.response.result
+              : null;
+        const methodOrEventName =
+          validationType == CONSTANTS.EVENT
+            ? apiOrEventObject.eventName
+            : validationType == CONSTANTS.METHOD
+              ? apiOrEventObject.apiName
+              : null;
+        // Loop through each item of validations array of validation object
+        validationTypeObject.validations.forEach((validation) => {
+          // If the field to be validated is result, directly assert if result stored in response is undefined
+          if (
+            validation.field === CONSTANTS.RESULT ||
+            validation.field === CONSTANTS.EVENT_RESPONSE
+          ) {
+            fireLog.isUndefined(
+              methodOrEventResponse,
+              `Undefined Validation : Expected ${methodOrEventName} response to have the field ${validation.field} as undefined`
+            );
+          } else {
+            // Else recursively access the nested field properties and verify the corresponding value in response is undefined
```

Comment:
Comment in line 44 should be below right ?
// Else recursively access the nested field properties and verify the corresponding value in response is defined